core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/PartitionQC.java

```diff
@@ -0,0 +1,121 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import uk.ac.bbsrc.tgac.miso.core.data.PartitionQC.PartitionQCId;
+import uk.ac.bbsrc.tgac.miso.core.data.impl.PartitionImpl;
+
+@Entity
+@Table(name = "Run_Partition_QC")
+@IdClass(PartitionQCId.class)
+public class PartitionQC implements Serializable {
+  public static class PartitionQCId implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @ManyToOne(targetEntity = PartitionImpl.class)
+    @JoinColumn(name = "partitionId")
+    private Partition partition;
+
+    @ManyToOne
+    @JoinColumn(name = "runId")
+    private Run run;
+
+    @Override
+    public boolean equals(Object obj) {
+      if (this == obj) return true;
+      if (obj == null) return false;
+      if (getClass() != obj.getClass()) return false;
+      PartitionQCId other = (PartitionQCId) obj;
+      if (partition == null) {
+        if (other.partition != null) return false;
+      } else if (!partition.equals(other.partition)) return false;
+      if (run == null) {
+        if (other.run != null) return false;
+      } else if (!run.equals(other.run)) return false;
+      return true;
+    }
+
+    public Partition getPartition() {
+      return partition;
+    }
+
+    public Run getRun() {
+      return run;
+    }
+
+    @Override
+    public int hashCode() {
+      final int prime = 31;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)